J&T Express

```diff
@@ -0,0 +1 @@
+J&T
```

Comment:
J&T Express

Reply:
@
@BRImo-id

Reply:
#10

Reply:
Duplicate of #

Reply:
#16

Reply:
https://github.com/smjltd/SMStoTelegram/pull/15#discussion_r1023755780